packages/pancake-uikit/src/components/Tag/types.ts

```diff
@@ -20,10 +20,11 @@ export const scales = {
 export type Scale = typeof scales[keyof typeof scales];
 export type Variant = typeof variants[keyof typeof variants];
 
-export interface TagProps extends SpaceProps {
+export interface TagProps extends SpaceProps, TypographyProps {
   variant?: Variant;
   scale?: Scale;
   startIcon?: ReactNode;
   endIcon?: ReactNode;
   outline?: boolean;
+  textTransform?: "uppercase" | "lowercase" | "capitalize";
```

Comment:
isn't it par of the `TypographyProps` helper already ?

Reply:
Surprisingly not. I thought it was but it seems you added it on the Text component.